src/sentry/tasks/auto_ongoing_issues.py

```diff
@@ -249,34 +255,61 @@ def auto_transition_issues_regressed_to_ongoing(
 @retry(on=(OperationalError,))
 @log_error_if_queue_has_items
 def auto_transition_issues_escalating_to_ongoing(
-    project_ids: List[int],
     date_added_lte: int,
-    project_id: Optional[int] = None,  # TODO(nisanthan): Remove this arg in next PR
     **kwargs,
 ) -> None:
-    # TODO(nisanthan): Remove this conditional in next PR
-    if project_id is not None:
-        project_ids = [project_id]
-
-    for new_groups in chunked(
+    """
+    We will update ESCALATING Groups to ONGOING that were created before the
+    most recent Group first seen 7 days ago. This task will trigger upto
+    50 subtasks to complete the update. We don't expect all eligible Groups
+    to be updated in a single run. However, we expect every instantiation of this task
+    to chip away at the backlog of Groups and eventually update all the eligible groups.
+    """
+    for new_group_ids in chunked(
         RangeQuerySetWrapper(
             Group.objects.filter(
-                project_id__in=project_ids,
                 status=GroupStatus.UNRESOLVED,
                 substatus=GroupSubStatus.ESCALATING,
                 grouphistory__status=GroupHistoryStatus.ESCALATING,
             )
             .annotate(recent_escalating_history=Max("grouphistory__date_added"))
             .filter(
                 recent_escalating_history__lte=datetime.fromtimestamp(date_added_lte, timezone.utc)
-            ),
+            )
+            .values_list("id", flat=True),
             step=ITERATOR_CHUNK,
+            limit=ITERATOR_CHUNK * 50,
+            result_value_getter=lambda item: item,
         ),
         ITERATOR_CHUNK,
     ):
-        bulk_transition_group_to_ongoing(
-            GroupStatus.UNRESOLVED,
-            GroupSubStatus.ESCALATING,
-            new_groups,
-            activity_data={"after_days": TRANSITION_AFTER_DAYS},
+        run_auto_transition_issues_escalating_to_ongoing.delay(
+            group_ids=new_group_ids,
         )
+
+
+@instrumented_task(
+    name="sentry.tasks.run_auto_transition_issues_escalating_to_ongoing",
+    queue="auto_transition_issue_states",
+    time_limit=25 * 60,
+    soft_time_limit=20 * 60,
+    max_retries=3,
+    default_retry_delay=60,
+    acks_late=True,
+    silo_mode=SiloMode.REGION,
+)
+@retry(on=(OperationalError,))
+def run_auto_transition_issues_escalating_to_ongoing(
```

Comment:
Optional feedback: This function and `run_auto_transition_issues_regressed_to_ongoing` are almost the same in case you want to join them and pass the status/substatus as parameters.